pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -334,6 +384,13 @@ func (exp *flowExporter) sendDataSet(dataSet ipfix.IPFIXSet, record flowexporter
 				// this dummy IP address.
 				ie.Value = net.IP{0, 0, 0, 0}
 			}
+		case "destinationClusterIPv6":
+			if record.Conn.DestinationServicePortName != "" {
+				ie.Value = record.Conn.TupleOrig.DestinationAddress
+			} else {
+				// Same as destinationClusterIPv4.
+				ie.Value = net.IP{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
nit: simplified version net.IP.Parse("::0")

Reply:
It seems `"::"` works as well. Updated.